compair/api/users.py

```diff
@@ -597,6 +601,132 @@ def get(self):
+            title="User's Third Party Logins Unavailable",
+            message="Sorry, your system role does not allow you to view third party logins for this user.")
+
+        lti_users = LTIUser.query \
+                .add_columns(
+                    LTIUser.uuid,
+                    LTIUser.compair_user_id,
+                    LTIUser.user_id,
+                    LTIUser.lti_consumer_id,
+                    LTIUser.lis_person_name_full,
+                    LTIConsumer.oauth_consumer_key,
+                    LTIConsumer.uuid.label("consumer_uuid")) \
+                .join(LTIConsumer) \
+                .filter(LTIUser.compair_user_id == user.id).all()
+
+        lti_users_list = [{
+                'uuid': t.uuid,
+                'lti_consumer_id': t.lti_consumer_id,
+                'lti_consumer_uuid': t.consumer_uuid,
+                'lti_user_id': t.user_id,
+                'oauth_consumer_key': t.oauth_consumer_key,
+                'compair_user_id': t.compair_user_id,
+                'lis_person_name_full': t.lis_person_name_full
+            } for t in lti_users]
+
+        on_user_lti_users_get.send(
+            self,
+            event_name=on_user_lti_users_get.name,
+            user=current_user,
+            data={'user_id': user.id})
+
+        return {"objects": lti_users_list}
+
+# /id/lti/users/uuid
+class UserLTIAPI(Resource):
+    @login_required
+    def delete(self, user_uuid, lti_user_uuid):
+        """
+        unlink lti user from compair user
+        """
+
+        user = User.get_by_uuid_or_404(user_uuid)
+
+        require(MANAGE, User,
+            title="User Unavailable",
+            message="Sorry, your system role does not allow you to unlink LTI users for this user.")
+
+        lti_user = LTIUser.get_by_uuid_or_404(lti_user_uuid)
+
+        if not lti_user:
+            abort(400, title="LTI User Not Unlinked", message="Sorry, The requested LTI user was not found.")
+
+        lti_user.compair_user_id = None
+        db.session.commit()
+
+        on_user_lti_user_unlink.send(
+            self,
+            event_name=on_user_lti_user_unlink.name,
+            user=current_user,
+            data={'user_id': user.id, 'lti_user_id': lti_user.id})
+
+        return { 'success': True }
+
+# /id/third_party/users
+class UserThirdPartyUserListAPI(Resource):
+    @login_required
+    def get(self, user_uuid):
+
+        user = User.get_by_uuid_or_404(user_uuid)
+
+        require(MANAGE, User,
+            title="User's Third Party Logins Unavailable",
+            message="Sorry, your system role does not allow you to view third party logins for this user.")
+
+        third_party_users = ThirdPartyUser.query.filter(ThirdPartyUser.user_id == user.id).all()
+
+        third_party_users_list = [{
+                'uuid': t.uuid,
+                'type': t.third_party_type.value,
+                'third_party_user_id': t.unique_identifier,
+                'compair_user_id': t.user_id
```

Comment:
`'compair_user_id': t.user_id` -> `'user_id': t.user_uuid` (send uuids instead of ids)